pkg/scheduler/util/utils_test.go

```diff
@@ -17,12 +17,57 @@ limitations under the License.
 package util
 
 import (
+	"fmt"
 	"reflect"
 	"testing"
+	"time"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/diff"
 	"k8s.io/kubernetes/pkg/apis/scheduling"
+	"k8s.io/kubernetes/pkg/scheduler/api"
+)
+
+var (
+	prio1, prio2 = int32(1), int32(2)
+
+	pod1 = &v1.Pod{
```

Comment:
It would be better to add them as variables in the function because test function might change them and it might effect other functions.

e.g. In `TestGetPodFullName`, the following is enough:

```go
pod := &v1.Pod {
	ObjectMeta: metav1.ObjectMeta{
		Namespace: "test",
		Name:      "pod",
	},
}
```

Reply:
Because different test functions would use these variables, maybe they can have a prefix in their name, eg, `testPod1`.

Reply:
I agree with @hex108's, we should avoid the package level variables.